app/controllers/support_interface/performance_controller.rb

```diff
@@ -1,9 +1,6 @@
 module SupportInterface
   class PerformanceController < SupportInterfaceController
-    skip_before_action :authenticate_support_user!, only: :index
-
     def index
```

Comment:
Would it make sense just to redirect to the new location?

Reply:
I'd like to add more stuff the the page in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1240, that's why we're keeping it and linking.